qutip/tests/core/test_qobjevo.py

```diff
@@ -131,6 +132,37 @@ def test_call(pseudo_qevo, coeff_type):
     assert not qevo.issuper
     _assert_qobjevo_equivalent(pseudo_qevo, qevo)
 
+# Test the QobjEvo.__repr__()
+def test_QobjEvo_repr():
+    # "case_n" are cases with different types of Objects of QobjEvo with unique __repr__
+    # "expected_repr_n" are the Expected result from the __repr__() of the QobjEvo objects
+
+    case_1= repr(QobjEvo([qeye(3), lambda t: t]))
+    expected_repr_1= 'QobjEvo: dims=[[3], [3]], shape=(3, 3), type=oper, superrep=None, isconstant=False, num_elements=1'
+    assert case_1 == expected_repr_1
+
+    case_2= repr(QobjEvo(qeye(2), lambda t: t))
```

Comment:
This case is strange since it's equivalent to `QobjEvo(qeye(2), args=lambda t: t)` and the `args` are not used...
It simply look as a badly formatted `QobjEvo`.